views/registrations/form.html

```diff
@@ -0,0 +1,23 @@
+<form class="form registration-form" action="/register" method="post">
```

Comment:
This violates the CSS convention to use the UI component as prefix

```suggestion
<form class="form form-registration" action="/register" method="post">
```

See [Compass](https://nimblehq.co/compass/development/code-conventions/css/#class-names):
> Use the UI element type as a prefix following by one to two words. While it might seem counter-intuitive at first, this technique makes it easier to differentiate component purpose and search through components.